README.md

```diff
@@ -20,48 +21,59 @@ function sum(a, b) {
   return a + b;
 } `); // define a function
 
-console.log(await vm.call("sum", 2, 3)) // 5
+console.log(await vm.call("sum", 2, 3)); // 5
 
 vm.close(); // terminate the worker
 ```
 
-Permissions
------------
+## Permissions
 
-* `--unstable` - permission options in worker is an unstable feature.
-* `--allow-read=path/to/worker.ts` - to launch the worker.
+### Default
 
-The code may terminate the worker
----------------------------------
+- `--unstable` - permission options in worker is an unstable feature.
+- `--allow-read=path/to/worker.ts` - to launch the worker.
 
-Although Deno is able to block disk/network access, we can't prevent the code from calling `self.close()`. If this happens, the worker will be terminated. All calls to `vm.run` will timeout.
+### Custom
 
-XXX is not cloneable
---------------------
+- `--allow-[read,write,net,env,sys,run,ffi,import]=...` - user can specify different permissions via `permissionOptions` to adapt to their use cases.
+
+## FAQ
+
+### The code may terminate the worker
+
+Although Deno is able to block disk/network access by default, we can't prevent the code from calling `self.close()`. If this happens, the worker will be terminated. All calls to `vm.run` will timeout.
+
+### XXX is not cloneable
 
 Data must be cloneable to be passed to the worker:
 
-* `VM.call` - function arguments are JSON-cloned.
-* The result of `VM.run` - throws an error if it is not cloneable.
-* `VM.on("console")` - function arguments are JSON-cloned.
+- `VM.call` - function arguments are JSON-cloned.
+- The result of `VM.run` - throws an error if it is not cloneable.
+- `VM.on("console")` - function arguments are JSON-cloned.
+
+### Requires net access to "example.com:443", run again with the --allow-net flag
```

Comment:
Remove `run again with the --allow-net flag`.

Underneath, explain that the permission of the worker can only be more strict than the parent thread.